crontabber/tests/test_crontabber.py

```diff
@@ -330,6 +370,11 @@ def test_pop(self):
         popped = self.database.pop('foo', 'default')
         eq_(popped, 'default')
         assert_raises(KeyError, self.database.pop, 'bar')
+        try:
+            del self.database['bar']
+            assert False, "it should have raise a KeyError on del"
+        except KeyError:
+            pass
```

Comment:
These are unitttest TestCase dervied, right? If so, you could use the context manager.

https://docs.python.org/2/library/unittest.html#unittest.TestCase.assertRaises

I don't remember if nose facilitates kind of thing or not. pytest has this:

http://pytest.org/latest/assert.html#assertions-about-expected-exceptions

Maybe nose has something similar.